Examples/Algorithms/Digitization/src/DigitizationConfig.cpp

```diff
@@ -62,7 +62,9 @@ makeSmearFunctionForType(SmearingTypes smearingType, const double* parameters) {
 ActsExamples::DigitizationConfig::DigitizationConfig(
     const Options::Variables& vars,
     Acts::GeometryHierarchyMap<DigiComponentsConfig>&& digiCfgs)
-    : isSimpleSmearer(vars["digi-smear"].as<bool>()) {
+    : isSimpleSmearer(vars["digi-smear"].as<bool>()),
+      doMerge(vars["digi-merge"].as<bool>()),
+      mergeNsigma(vars["digi-merge-nsigma"].as<double>()) {
```

Comment:
Again, for geometric clustering, we should not merge on sigma, but on connectivity.

Reply:
See above